<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConflictNonExpert</class>
 <widget class="QDialog" name="DialogConflictNonExpert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration conflict</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>The current values in the driver board do not match the IcePAP database</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>The driver board is either new or has been moved to a different slot.

The current values in the driver board will be lost.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMoreInfo">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>100</width>
     <height>26</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Show values</string>
   </property>
   <property name="text">
    <string>More info ...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUpdate">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>100</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Send database config
to driver board</string>
   </property>
   <property name="text">
    <string>Update driver</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>100</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Cancel driver
configuration</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
